code/modules/telesci/bscrystal.dm

```diff
@@ -46,7 +46,7 @@
 
 // Polycrystals, aka stacks
 
-var/global/list/datum/stack_recipe/bluespace_crystal_recipes = list(new/datum/stack_recipe("Breadkdown into bluespace crystal", /obj/item/weapon/ore/bluespace_crystal, 1, one_per_turf = 0, on_floor = 1))
+var/global/list/datum/stack_recipe/bluespace_crystal_recipes = list(/datum/stack_recipe/bluespacecrystals("Breakdown into bluespace crystal", /obj/item/weapon/ore/bluespace_crystal, 1, one_per_turf = 0, on_floor = 1))
```

Comment:
You also want to put that `new` back, so there'll be an actual instance of this `stack_recipe` to call